src/main/java/calculator/view/InputView.java

```diff
@@ -0,0 +1,16 @@
+package calculator.view;
+
+import java.util.Scanner;
+
+public class InputView {
+    private static final Scanner scanner = new Scanner(System.in);
+    private static final String PRINT_INPUT_FORMULA = "계산할 숫자와 연산자를 입력해주세요 : ";
```

Comment:
```suggestion
    private static final String PRINT_INPUT_FORMULA = "계산할 숫자와 연산자를 입력해주세요 : ";
    
```
필드와 메서드를 구분하기 위해서 줄바꿈을 추가해도 좋을 것 같습니다.